.gitignore

```diff
@@ -8,10 +8,10 @@ logs/
 dist/
 firefox/
 *.update.rdf
-system-addon/data/content/activity-stream.bundle.js
-system-addon/data/content/*.js.map
-system-addon/data/locales.json
-system-addon/css/
-system-addon/prerendered/
+data/content/activity-stream.bundle.js
+data/content/*.js.map
+data/locales.json
```

Comment:
I noticed `locales.json` being copied over, but I was initially confused why it didn't show up in mozilla-central's `git status`, and it's because m-c git recursively looks at sub `.gitignore`s. We happen to be okay with existing checked-in-to-m-c files, but at least for right now, just because we don't want to check something in to activity-stream repository doesn't mean we also want to ignore it in mozilla-central, e.g., `prerendered/` files.

It does look like the only ignored and untracked file is `locales.json`, so maybe we just make sure that's added to `.mcignore`?

Reply:
I think probably the right thing to do here is not to copy over the `.gitignore`